plugin/azure/README.md

```diff
@@ -13,7 +13,7 @@ record types.
 ## Syntax
 
 ~~~ txt
-azure RESOURCE_GROUP:ZONE... {
+azure RESOURCE_GROUP:ZONE:ACCESS... {
```

Comment:
I would put this in the options below, by default it's public, unless you have `private` as an option

Reply:
In that case, there will have to be a separate azure block for private and public zones. Currently it looks like this - https://github.com/coredns/coredns/pull/3516/files#diff-694fec5c821a1b8f941cbb681c604bdeR45

What do you think?

Reply:
but then just allow 'private, public and both' as an option?

Reply:
sorry I did not fully understand.
Are you suggesting - having something like:

```
    azure resource_group_foo:example.org:both {
      tenant 123abc-123abc-123abc-123abc
      client 123abc-123abc-123abc-234xyz
      subscription 123abc-123abc-123abc-563abc
      secret mysecret
    }
```

or

```
    azure resource_group_foo:example.org resource_group_foo:example.private {
      tenant 123abc-123abc-123abc-123abc
      client 123abc-123abc-123abc-234xyz
      subscription 123abc-123abc-123abc-563abc
      secret mysecret
      access both
    }
```

Or neither? 

`azure-sdk-for-go` treats the public zones and private zones as two distinct and disjoint types, with different API to interact with them and hence we need to identify which is which. So, both the above won't work.

Reply:
the latter, have a property the signals the access level, access [private|both] make sense and defaulting to public (which isn't an option for access because it is the default)